tests/snapshots/tests.contrib.azure_functions.test_azure_functions_snapshot.test_http_get_error.json

```diff
@@ -0,0 +1,37 @@
+[[
+  {
+    "name": "azure.functions.invoke",
+    "service": "azure_functions",
+    "resource": "GET /api/httpgeterror",
+    "trace_id": 0,
+    "span_id": 1,
+    "parent_id": 0,
+    "type": "serverless",
+    "error": 1,
+    "meta": {
+      "_dd.base_service": "",
+      "_dd.p.dm": "-0",
+      "_dd.p.tid": "6750ad8500000000",
+      "aas.function.name": "http_get_error",
+      "aas.function.trigger": "Http",
+      "component": "azure_functions",
+      "error.message": "Test Error",
+      "error.stack": "Traceback (most recent call last):\n  File \"/root/project/ddtrace/contrib/internal/azure_functions/patch.py\", line 65, in wrap_function\n    res = func(req)\n          ^^^^^^^^^\n  File \"/root/project/tests/contrib/azure_functions/azure_function_app/function_app.py\", line 19, in http_get_error\n    raise Exception(\"Test Error\")\nException: Test Error\n",
+      "error.type": "builtins.Exception",
+      "http.method": "GET",
+      "http.route": "/api/httpgeterror",
+      "http.url": "http://0.0.0.0:7071/api/httpgeterror",
+      "http.useragent": "python-httpx/x.xx.x",
+      "language": "python",
+      "runtime-id": "d7efb82603894b91af0e18f95bfb40ce",
+      "span.kind": "server"
```

Comment:
Is it expected that the span.type is "serverless" here but the span tag is "span.kind: server"?

Reply:
That's expected, see https://github.com/DataDog/dd-trace-py/pull/11474#discussion_r1880648337